### PR TITLE
makes tesla not favor the east as much i think idk tbh fam ttyl

### PR DESCRIPTION
**Pull request by Iamgoofball**
posi vibes

### CODE REVIEW

File: code/modules/power/tesla/energy_ball.dm [hunk 1]
```diff
@@ -81,10 +81,8 @@ var/list/blacklisted_tesla_types = typecacheof(list(/obj/machinery/atmospherics,
 
 
 /obj/singularity/energy_ball/proc/move_the_basket_ball(var/move_amount)
-	//we face the last thing we zapped, so this lets us favor that direction a bit
-	var/first_move = dir
 	for(var/i in 0 to move_amount)
-		var/move_dir = pick(alldirs + first_move) //give the first move direction a bit of favoring.
+		var/move_dir = pick(alldirs)
```
Comment (RemieRichards) [inline code/modules/power/tesla/energy_ball.dm]: put it back, and do ```setDir(move_dir)``` below the ```loc = T```, the issue is that ```loc = T``` doesn't set dir

Comment (Iamgoofball) [inline code/modules/power/tesla/energy_ball.dm]: sik

**Comment by RemieRichards** [comment]
also do it where tesla_zap() is called (using the return value of tesla_zap()) as that's what the comment implies.

**Comment by Iamgoofball** [comment]
fugggg

**Comment by Iamgoofball** [comment]
already done at line 67

**Comment by RemieRichards** [comment]
oh right I was looking at the balls, sure.

**Comment by Iamgoofball** [comment]
>I was looking at the balls

**Comment by PKPenguin321** [comment]
If this works you're a god

**Comment by calzilla1** [comment]
Iamgodball

**Comment by Haevacht** [comment]
I thought this was just a meme, but it, is the impossible, goof fixing his shit. They said it couldn't be done.

**Pull request merged**